tools/apidiff/merger.cs

```diff
@@ -76,22 +84,29 @@ public static void Process (string platform, string path, string os)
 		headers.WriteLine ($"# {title}");
 		headers.WriteLine ();
 
-		File.WriteAllText (filename, headers.ToString ());
+		var filePath = DestinationPath is not null ? Path.Combine (DestinationPath, filename) : filename;
+		File.WriteAllText (filePath, headers.ToString ());
 
 		var alldiffs = content.ToString ();
 		if (alldiffs.Length == 0)
 			alldiffs = "No changes were found between both versions."; // should not happen for releases (versions change)
-		File.AppendAllText (filename, alldiffs);
-		Console.WriteLine ($"@MonkeyWrench: AddFile: {Path.GetFullPath (filename)}");
+		File.AppendAllText (filePath, alldiffs);
+		Console.WriteLine ($"@MonkeyWrench: AddFile: {Path.GetFullPath (filePath)}");
 
-		if (File.Exists ("api-diff.html"))
-			File.AppendAllText ("api-diff.html", $"\n<h2><a href=\"{filename}\">{platform} API diff (markdown)</a></h2>");
+		if (File.Exists ("api-diff.html")){
```

Comment:
```suggestion
		if (File.Exists ("api-diff.html")) {
```

Reply:
I also don't see the need for this {} wrapping, it doesn't seem like you are making any other changes here.

Reply:
Is the filename, the diff is misleading due to the added {}. @tj-devel709 this is an example of why we should just change what is needed ella you are making it hard for the reviewers

Reply:
You're right! I had debugging here before and forgot to remove brackets!